api/controllers/SdtdServer/server-tile.js

```diff
@@ -38,16 +38,25 @@ module.exports = {
     unknownError: {
       responseType: 'badRequest'
     },
+    redirectTemporary: {
+      responseType: 'redirectTemporary'
+    }
   },
 
   fn: async function (inputs, exits) {
-    let server = await SdtdServer.findOne(inputs.serverId);
+    const server = await SdtdServer.findOne(inputs.serverId).populate('config');
+    // is there a cheap way to call api/helpers/user/get-servers-with-permission.js
```

Comment:
Why? :)

Reply:
it was thinking of something cheaper than the policies, but yea, meh